tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -144,7 +143,7 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
                     final CauseAction causeAction = new CauseAction(cause);
                     final Action[] actionArray = ActionHelper.create(actionsWithSafeParams, causeAction);
                     scmTriggerItem.scheduleBuild2(quietPeriod, actionArray);
-                    if (gitCodePushedEventArgs instanceof PullRequestMergeCommitCreatedEventArgs) {
+                    if (gitCodePushedEventArgs instanceof PullRequestMergeCommitCreatedEventArgs && !gitCodePushedEventArgs.collectionUri.toString().equalsIgnoreCase(TESTING_COLLECTION)) {

```

Comment:
I don't understand the need to exclude the "TESTING_COLLECTION" here

Reply:
The exclusion is necessary as TESTING_COLLECTION is a fake collection and its url isn't accessible. When sendPullRequestBuildStatusEvent tries to query TESTING_COLLECTION it fails due to auth issue. Excluding it as we don't need to actually reach the testing collection.

Reply:
Actually I changed to update queue status in a different location. Now we no longer need "TESTING_COLLECTION", and I've hence removed it.